spec/lib/ulid_spec.rb

```diff
@@ -103,4 +103,31 @@
       end
     end
   end
+
+  describe 'decoding a timestamp' do
```

Comment:
ULIDs are case-insensitive. This currently fails:

```ruby
it 'decodes in a case-insensitve manner' do
  ulid = ULID.generate.downcase
  assert ULID.decode_time(ulid).is_a?(Time)
end
```

For completeness, I'd like to see a few known input/output pairs (but please don't feel obliged to include them):

```ruby
it 'correctly parses known test vectors' do
  {
    "0000000000Y2GBSG3FEFT635J1" => Time.at(0),
    "013XRZP292318JWRM98F0YAPV9" => Time.at(1234567891234/1000r),
    "ZZZZZZZZZZ-this-is-ignored" => Time.at(1125899906842623/1000r),
  }.each do |vector, expected_time|
    assert_equal expected_time, ULID.decode_time(vector)
  end
end
```